mon-pix/app/templates/terms-of-service.hbs

```diff
@@ -0,0 +1,39 @@
+<div class="terms-of-service-page">
+  <div class="terms-of-service-form">
+
+    <div class="terms-of-service-form__text">
+      <a href={{urlHome}} class="pix-logo__link" title="Page d'accueil">
+        <img class="pix-logo__image" src="{{rootURL}}/images/pix-logo.svg" alt="Pix">
+      </a>
+    </div>
+
+    <div class="terms-of-service-form__text">
+      <h2>Conditions d'utilisation</h2>
+    </div>
+
+    <div class="terms-of-service-form__text">
+      Nous avons mis à jour nos conditions d'utilisation.
+    </div>
+
+    <div class="terms-of-service-form__text">
+      <div class="terms-of-service-form__conditions">
+        <label for="pix-cgu" >
+          <div>
+            <Input @type="checkbox" @id="pix-cgu" class="terms-of-service-form__check"/>
+            J'accepte les <a href="https://pix.fr/conditions-generales-d-utilisation" class="link" target="_blank">conditions
+            d'utilisation de Pix</a>
+          </div>
+        </label>
+      </div>
+    </div>
+    <div class="terms-of-service-form__text">
```

Comment:
Un peu bizarre de voir `terms-of-service-form__text` à cette endroit, je verrai un truc plutôt du genre : `terms-of-service-form__actions`

Reply:
Done